src/app/components_ngrx/infotip/infotip.component.ts

```diff
@@ -1,12 +1,12 @@
-import { Component, Input, HostListener } from '@angular/core';
+import { Component, Input} from '@angular/core';
 
 @Component({
   selector: 'infotip',
-  template: `<template #popoverTemplate>
+  template: `<ng-template #popoverTemplate>
```

Comment:
make HTML template for infotips

Reply:
Also, infotips css is not here. Please add it here as template specific CSS rather than in assets folder